test/externalTests/perpetual-pools.sh

```diff
@@ -64,6 +64,10 @@ function perpetual_pools_test
     sed -i 's|\(it\)\(("Should not allow commits that are too large"\)|\1.skip\2|g' test/PoolCommitter/commit.spec.ts
     sed -i 's|\(it\)\(("Should not allow for too many commitments (that bring amount over a user'\''s balance)"\)|\1.skip\2|g' test/PoolCommitter/commit.spec.ts
 
+    # Disable a test failing due to a non-deterministic order of keys in a returned dict.
+    # TODO: Figure out why it's failing and re-enable.
+    sed -i 's|\(it\)\(("Rotates the observations array"\)|\1.skip\2|g' test/PriceObserver.spec.ts
```

Comment:
Here's the relevant output from the [the failed run of `t_native_test_ext_perpetual_pools`](https://app.circleci.com/pipelines/github/ethereum/solidity/33176/workflows/bb77f2ae-141b-49ed-b355-bbf91188d51a/jobs/1491324):

```
  262 passing (10m)
  15 pending
  1 failing

  1) PriceObserver
       add
         When called with a full observations array
           Rotates the observations array:

      AssertionError: expected [ …(24) ] to deeply equal [ …(24) ]
      + expected - actual

       [
         {
      +    "_hex": "0x03"
      +    "_isBigNumber": true
      +  }
      +  {
           "_hex": "0x04"
           "_isBigNumber": true
         }
         {
--
           "_hex": "0x07"
           "_isBigNumber": true
         }
         {
      +    "_hex": "0x08"
      +    "_isBigNumber": true
      +  }
      +  {
           "_hex": "0x0c"
           "_isBigNumber": true
         }
         {
      -    "_hex": "0x08"
      +    "_hex": "0x0a"
           "_isBigNumber": true
         }
         {
           "_hex": "0x0b"
           "_isBigNumber": true
         }
         {
      -    "_hex": "0x0a"
      +    "_hex": "0x0c"
           "_isBigNumber": true
         }
         {
           "_hex": "0x0e"
--
           "_hex": "0x05"
           "_isBigNumber": true
         }
         {
      -    "_hex": "0x03"
      +    "_hex": "0x05"
           "_isBigNumber": true
         }
         {
           "_hex": "0x09"
           "_isBigNumber": true
         }
         {
      -    "_hex": "0x0c"
      -    "_isBigNumber": true
      -  }
      -  {
      -    "_hex": "0x05"
      -    "_isBigNumber": true
      -  }
      -  {
           "_hex": "0x0a"
           "_isBigNumber": true
         }
         {
```